.vscode/launch.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
I'm not sure how I feel about the `.vscode` files being added, to be frank. I don't hate this in any sense, but, as I don't use this editor (if you can't tell from my atrocious spelling abilities), I'm not sure I want to be responsible for maintaining this.

Is this something that should absolutely be included in the repository, or could you simply hold on to these files?

@tfoote do you have any opinions on this? I'm not just not sure this is best practice.

Reply:
I can definitely remove it. I thought that would be helpful for other people and non-intrusive but it's totally fine to carry it myself.

Reply:
Yeah, in general we try to avoid development tool specific as it's both a personal choice of what editor to use as well as how you want to set it up.

Reply:
Thanks, I've dropped the .vscode files.